### ISSUE TITLE
Make 8.x checks a parameter of the action

### ISSUE THREAD

**Issue by pwtyler** [pantheon-systems/phpcompatibility-action#8]
Instead of using an @dev branch whose only difference is using an unreleased version of the upstream, make the option for 8.x checks an input parameter of running the action

**Comment by benoitchantre** [comment]
I don’t understand why the `test-versions` defaults to `8.0-`, but you need to use the @dev branch to benefit from appropriate dependencies.

@pwtyler What would be the downsides to merge the `develop` branch to `main`?

**Comment by pwtyler** [comment]
I assume @kporras07's intention was to not have the "stable" branch of the action point to an unstable version of the PHPCompatibility project, but since we're at 4+ years since a release, it's likely the right choice to just default to using the `develop` branch of PHPCompatibility.

**Comment by kporras07** [comment]
Yeah, also I think I initially didn't notice that you needed dev to get those versions. I'm happy with the decision that @pwtyler makes here :)

**Comment by pwtyler** [comment]
It will be easier to make develop the default state rather than trying to parameterize using the development version or not. With 7.4 long-EOL, I expect the vast majority of use cases will be PHP 8+, so will just be making the "breaking" change.
